### PR TITLE
fix: PyMuPDF Reader broken

### PR DESCRIPTION
**Pull request by EmanuelCampos**
# Description

PyMuPDF with SimpleDirectoryReader `file_extractor` wasn't working since it uses `load_data()` instead of `load()`

Fixes https://github.com/emptycrown/llama-hub/issues/509

## Type of Change

Please delete options that are not relevant.

- [ ] New Loader/Tool
- [x] Bug fix / Smaller change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [x] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [ ] I have added a library.json file if a new loader/tool was added
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: llama_hub/file/pymu_pdf/base.py [hunk 1]
```diff
@@ -9,6 +9,12 @@
 class PyMuPDFReader(BaseReader):
     """Read PDF files using PyMuPDF library."""
 
+    def load_data(
+        self, file_path: Union[Path, str], extra_info: Optional[Dict] = None
+    ) -> List[Document]:
+        """Loads list of documents from PDF file and also accepts extra information in dict format."""
+        return self.load(file_path, metadata=True, extra_info=extra_info)
+
     def load(
```
Comment (logan-markewich) [inline llama_hub/file/pymu_pdf/base.py:18]: I wonder if we should just remove the load function -- I feel like it technically shouldn't even be there 😅

File: llama_hub/file/pymu_pdf/base.py
Comment (logan-markewich) [inline llama_hub/file/pymu_pdf/base.py:1]: We should also update the example in the readme 
https://llamahub.ai/l/file-pymu_pdf

File: llama_hub/file/pymu_pdf/base.py (hunk 1 above)
Comment (EmanuelCampos) [reply llama_hub/file/pymu_pdf/base.py:18]: I just kept it for backwards compatibility

File: llama_hub/file/pymu_pdf/base.py
Comment (EmanuelCampos) [reply llama_hub/file/pymu_pdf/base.py:1]: fixed

**Pull request merged**